fury/actor.py

```diff
@@ -1593,7 +1593,11 @@ def sphere(centers, colors, radii=1., phi=16, theta=16,
         sphere_actor.GetProperty().SetOpacity(opacity)
         return sphere_actor
 
-    scales = np.multiply(radii, [1, 1, 1])
+    if type(radii) in [float, int]:
+        scales = np.array([1, 1, 1]) * radii
+    else:
+        scales = np.array([1, 1, 1]) * radii[:, None]
```

Comment:
This works perfectly fine @m-agour , 
You may also use `scales = np.array(radii)` as `repeat_primitive` handles the reshaping of the scales array.

Reply:
Hello @xtanion,
Thank you for reviewing my PR.
The compact solution you mentioned works just fine. Thanks for mentioning it.